dev-docs/bidder-adaptor.md

```diff
@@ -354,6 +355,12 @@ Some of the data in `ortb2` is also made available through other `bidderRequest`
 * **gdprConsent** is the object containing data from the [GDPR ConsentManagement](/dev-docs/modules/consentManagement.html) module. For TCF2+, it will contain both the tcfString and the addtlConsent string if the CMP sets the latter as part of the TCData object.
 * **uspConsent** is the object containing data from the [US Privacy ConsentManagement](/dev-docs/modules/consentManagementUsp.html) module.
 
+<a id="tid-warning" />
```

Comment:
```suggestion
<a id="tid-warning"></a>
```

`<a>` cannot be self closing. https://developer.mozilla.org/en-US/docs/Glossary/Void_element